digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java

```diff
@@ -1171,7 +1171,26 @@ public long getActiveAttemptCount()
         @Override
         public List<StoredSessionWithLastAttempt> getSessions(int pageSize, Optional<Long> lastId)
         {
-            return autoCommit((handle, dao) -> dao.getSessions(siteId, pageSize, lastId.or(Long.MAX_VALUE)));
+            return getSessions(pageSize, lastId, 1);
+        }
+
+        @Override
+        public List<StoredSessionWithLastAttempt> getSessions(int pageSize, Optional<Long> lastId, int pageNumber)
```

Comment:
I guess switch the order of pageNumber and lastId may be understandable.
Other method also same.